### PR TITLE
docs: Replace Slack with Discord

### PR DESCRIPTION
**Pull request by macolo**
## Description

- feature Discord as the main comms channel across all django CMS documentation

## Checklist

* [x] I have opened this pull request against ``develop-4``
* [ ] I have added or modified the tests when changing logic
* [ ] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [ ] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on [Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Review by fsbraun — APPROVED** [review]
LGTM!

File: docs/contributing/code.rst [hunk 1]
```diff
@@ -17,7 +17,7 @@ Here's what the contribution process looks like in brief:
 #. Fork our `GitHub`_ repository, https://github.com/django-cms/django-cms
 #. Work locally and push your changes to your repository.
 #. When you feel your code is good enough for inclusion, send us a pull request.
-#. After that, please join the `Slack Channel`_ of our Pull Request Review work group (#workgroup-pr-review). This group of friendly community members is dedicated to reviewing pull requests. Report your PR and find a “pr review buddy” who is going to review your pull request.
+#. After that, please join our Discord server (`#contributors <https://discord.gg/mwRxJSBP>`_). This group of friendly community members is dedicated to reviewing pull requests. Report your PR and find a “pr review buddy” who is going to review your pull request.
```
Comment (marksweb) [inline docs/contributing/code.rst:20]: Is this an unlimited link?

**Review by marksweb — COMMENTED** [review]
I don't really understand discord links but I know they can have time limits. And I note that there's various different links here, so just want to make sure we're happy that they're not likely to expire.

**Comment by fsbraun** [comment]
@macolo @marksweb I have replaced all discord links by `https://www.django-cms.org/discord` we can manage all redirects there then. For now, this redirects to `https://discord.com/channels/800813886689247262/1201999613272535130` which to my understanding is a permanent link. If not, we only need to change the redirect.

**Pull request merged**